sdk/core/Azure.Core/tests/HttpClientTransportFunctionalTest.cs

```diff
@@ -18,23 +26,31 @@ static HttpClientTransportFunctionalTest()
 #if NET461
             ServicePointManager.ServerCertificateValidationCallback += (_, _, _, _) => true;
 #endif
+
         }
+
         public HttpClientTransportFunctionalTest(bool isAsync) : base(isAsync)
-        {
-        }
+        { }
 
-        protected override HttpPipelineTransport GetTransport(bool https = false)
+        protected override HttpPipelineTransport GetTransport(bool https = false, HttpPipelineTransportOptions options = null)
         {
             if (https)
             {
 #if !NET461
-                return new HttpClientTransport(new HttpClientHandler()
+                return options switch
                 {
-                    ServerCertificateCustomValidationCallback = HttpClientHandler.DangerousAcceptAnyServerCertificateValidator
-                });
+                    null => new HttpClientTransport(
```

Comment:
We should use HttpPipelineTransportOptions to suppress instead. That way we won't replace the default handler.